### PR TITLE
DM:-15073 Add datasets to support TransformObjectCatalogTask

### PR DESCRIPTION
**Pull request by yalsayyad**

### CODE REVIEW

File: policy/datasets.yaml [hunk 1]
```diff
@@ -1119,4 +1119,27 @@ deepCoadd_sg_features_tract:
     storage: ParquetStorage
     python: lsst.qa.explorer.parquetTable.ParquetTable
     template: deepCoadd-results/merged/%(tract)d/sg-feats-%(tract)d.parq
-
+transformObjectCatalog_config:
+    persistable: Config
+    python: lsst.qa.explorer.postprocess.TransformObjectCatalogConfig
+    storage: ConfigStorage
+    template: config/TransformObjectCatalog
+consolidateObjectTable_config:
```
Comment (TallJimbo) [inline policy/datasets.yaml]: Description for this and the one above would be welcome.  I imagine I'll learn what they are looking at obs_subaru, but for now I'm curious.

File: policy/datasets.yaml [hunk 2]
```diff
@@ -1119,4 +1119,31 @@ deepCoadd_sg_features_tract:
...
     python: lsst.qa.explorer.parquetTable.ParquetTable
     template: deepCoadd-results/merged/%(tract)d/sg-feats-%(tract)d.parq
-
+transformObjectCatalog_config:
+    description: >
+        Configuration for Task to transform deepCoadd_obj table into objectTable
+    persistable: Config
+    python: lsst.qa.explorer.postprocess.TransformObjectCatalogConfig
+    storage: ConfigStorage
+    template: config/TransformObjectCatalog.py
```
Comment (laurenam) [inline policy/datasets.yaml]: Should the “T” really be capitalized?

File: policy/datasets.yaml [hunk 3]
```diff
@@ -1119,4 +1119,31 @@ deepCoadd_sg_features_tract:
...
+    storage: ParquetStorage
+    python: lsst.qa.explorer.parquetTable.MultilevelParquetTable
+    template: deepCoadd-results/merged/%(tract)d/%(patch)s/ObjectTable-%(tract)d-%(patch)s.parq
+objectTable_tract:
+    description: >
+        Per-tract DPDD Object Table
+    persistable: ignored
+    storage: ParquetStorage
+    python: lsst.qa.explorer.parquetTable.MultilevelParquetTable
+    template: deepCoadd-results/merged/%(tract)d/ObjectTable-%(tract)d.parq
```
Comment (laurenam) [inline policy/datasets.yaml]: Ditto for the “O”s

File: policy/datasets.yaml [hunk 4]
```diff
@@ -1119,4 +1119,31 @@ deepCoadd_sg_features_tract:
...
+consolidateObjectTable_config:
+    description: >
+        Configuration for Task to concatenate objectTables
+    persistable: Config
+    python: lsst.qa.explorer.consolidateQATable.ConsolidateObjectTableConfig
+    storage: ConfigStorage
+    template: config/consolidateObjectTable.py
+objectTable:
+    description: >
+        Per-Patch DPDD Object Table computed from the deepCoadd_obj catalog.
```
Comment (laurenam) [inline policy/datasets.yaml]: Period here after description, but not on others.

**Pull request merged**